README.md

```diff
@@ -6,126 +6,98 @@ This is a FastAPI application that demonstrates basic CRUD operations with a SQL
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── init.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── payment_method.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── product_category.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;└── region.py<br>
+│ └── models/ <br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── init.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;└── models.py<br>
+│ └── routers/ <br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── init.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;└── endpoints.py<br>
+│ └── utils/ <br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── init.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;├── config.py<br>
+│ &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;└── logger_config.py<br>
 │ ├── init.py<br>
 │ ├── crud.py<br>
-│ ├── models.py<br>
 │ ├── schemas.py<br>
-│ ├── database.py<br>
-│ └── routers/<br>
-│     ├── init.py<br>
-│     └── items.py<br>
-└── main.py
+│ └── database.py<br>
+├── database/<br>
+│ └── Online Sales Data.csv <br>
+├── docker/<br>
+│ └── docker-compose.yml <br>
+├── process_csv.py <br>
+├── .env.template <br>
+└── main.py <br>
 
 
-## Requirements
-
-- Python 3.8+
-- FastAPI
-- SQLAlchemy
-- Uvicorn
-- Pydantic
-
-## Installation
-
-
-- **main.py**: Main application file that initializes FastAPI and includes routers, Entry point to start the application with Uviorn.
-- **app/database.py**: Database configuration and session management.
-- **app/models.py**: SQLAlchemy models.
-- **app/schemas.py**: Pydantic schemas for data validation.
-- **app/crud.py**: CRUD operation functions.
-- **app/routers/items.py**: API endpoints for items.
-
-## Setup and Installation
+## Getting Started
 
 ### Prerequisites
 
-- Python 3.7+
-- SQLite (or another supported database)
-
-### Installation Steps
+- [Docker](https://www.docker.com/get-started)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 
-1. **Clone the repository:**
+### Installation
 
-    ```bash
+1. Clone the repository:
+    ```sh
     git clone https://github.com/StiVit/CAUTIOUS-WADDLE.git
     cd CAUTIOUS-WADDLE
     ```
 
-2. **Create and activate a virtual environment:**
-
-    ```bash
-    python -m venv venv
-    source venv/bin/activate  # On Windows, use `venv\Scripts\activate`
-    ```
-
-3. **Install the dependencies:**
-
-    ```bash
-    pip install -r requirements.txt
-    ```
-
-4. **Run the application:**
-
-    ```bash
-    python run.py
-    ```
-
-    The application will be available at `http://127.0.0.1:8000`.
-
-## Usage
-
-### API Endpoints
-
-- **Create an item**
-
-    ```http
-    POST /api/items/
+2. Create a `.env` file from the `.env.template`:
+    ```sh
+    cp .env.template .env
```

Comment:
Good stuff. However I think you should still keep the part of documentation where you show how to activate venv and install dependencies. If you want to automate it throughly, you can search about `makefile` and how to work with them. This will be the best solution I think ;)